ci/Jenkinsfile.combined

```diff
@@ -55,8 +62,14 @@ pipeline {
       steps { script {
         e2eUrl = cmn.uploadArtifact(findFiles(glob: 'pkg/*e2e.apk')[0].path)
         apkUrl = cmn.uploadArtifact(findFiles(glob: "pkg/*${cmn.getBuildType()}.apk")[0].path)
-        dmgUrl = cmn.uploadArtifact(findFiles(glob: 'pkg/*.dmg')[0].path)
-        appUrl = cmn.uploadArtifact(findFiles(glob: 'pkg/*.AppImage')[0].path)
+
+        if (cmn.getBuildType() != 'release') {
+          dmgUrl = cmn.uploadArtifact(findFiles(glob: 'pkg/*.dmg')[0].path)
+          appUrl = cmn.uploadArtifact(findFiles(glob: 'pkg/*.AppImage')[0].path)
+        } else {
+          dmgUrl = '-'
+          appUrl = '-'
```

Comment:
If you set those as `-` then the `misc/status-im.github.io-update_env` job will run with that as the parameter values and set those to be the links on the https://status-im.github.io/nightly/ page.

Reply:
the values should be `null` or empty string. Then the values inside of https://github.com/status-im/status-im.github.io/blob/develop/env.sh will not be changed.

Reply:
you can't really get them there because we don't update the website for release builds, but point taken, I'll tweak it.

Reply:
Oh, good point, forgot Publish runs only for nightlies.